guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc

```diff
@@ -59,6 +59,10 @@ ifdef::satellite[]
 ----
 # {foreman-maintain} content prepare
 ----
++
+. When the `satellite-maintain content prepare` step is complete {Project} checks whether any content units are unmigrated.
```

Comment:
Please use variables at all times

Reply:
This. I wonder if we could somehow automatically catch this with some automated check in the future so we don't depend on human review as much.